### ISSUE TITLE
:unused-namespaces false positive when implementing a protocol

### ISSUE THREAD

**Issue by dpassen** [jonase/eastwood#162]
Simply requiring a namespace for a protocol to be implemented in a record causes a false positive when using the :unused-namespaces linter.

```
lein eastwood '{:linters [:unused-namespaces]}' 
== Eastwood 0.2.1 Clojure 1.6.0 JVM 1.7.0_51
Directories scanned for source files:
  src test
== Linting protocol-test.foo ==
== Linting protocol-test.bar ==
Entering directory `/Users/derek/apps/protocol-test'
src/protocol_test/bar.clj:1:1: unused-namespaces: Namespace protocol-test.foo is never used in protocol-test.bar
== Warnings: 1 (not including reflection warnings)  Exceptions thrown: 0
Subprocess failed
```

bar.clj

```
(ns protocol-test.bar
  (:require [protocol-test.foo :as foo]))

(defrecord Bar [whatever]
  foo/Foo
  (foo [this] whatever))
```

foo.clj

```
(ns protocol-test.foo)

(defprotocol Foo
  (foo [this]))
```

**Comment by jafingerhut** [comment]
Thanks for the small test case to reproduce the issue.

**Comment by bencripps** [comment]
@jafingerhut ugh i was bit by this one too, any chance of getting this merged?

**Comment by abtv** [comment]
Any updates on it? @jafingerhut

**Comment by jafingerhut** [comment]
This should be fixed in the forthcoming Eastwood 0.2.5 release.
